ep/src/lib.rs

```diff
@@ -1,13 +1,15 @@
-use psm::epid::*;
-use psm::error::*;
-use psm::consts::*;
+extern crate error;
+use error::*;
```

Comment:
In this instance im using 2/3 things in the error crate, but most of the time fns will only need to return an error code. So ill start importing what i need at the moment.